index.d.ts

```diff
@@ -58,9 +254,267 @@ export type SupportedLanguages =
 	| "ko"
 	| "nl"
 	| "pl"
+	| "pt"
 	| "pt-br"
 	| "ru"
+	| "tlh"
 	| "tr"
+	| "uk"
+	| "uk-ua"
+	| "zh"
 	| "zh-cn"
-	| "zh-tw"
-	| "uk";
+	| "zh-tw";
+
+/**
+ * ISO 3166-1 Alpha-2 codes
+ */
+export type Iso3166Alpha2Code =
```

Comment:
These are [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), not language codes (which is [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))

We already have our `SupportedLanguages` union, so combining it with a full list of existing languages would only cause confusion which languages are truly supported.

The language is really only ever overriden with a static string (eg your site is German and you only expect German visitors, you force it to German), so allowing every possible language in here might cause site owners to set it to a language that is not yet supported. (it may be supported it in the future, but `turnstile-types` will just update in that case).  
The only reason to accept every language I can see is where the language is taken from a user-defined source (eg `navigator.language`), but in that case I'd either let turnstile auto-detect the language or use a `as SupportedLanguage` type-cast if you absolutely want to.